docs/source/research.rst

```diff
@@ -0,0 +1,72 @@
+.. mitiq documentation file
+
+.. _research:
+
+========
+Research
+========
+Mitiq is designed in order to aid researchers in quantum computing and quantum error mitigation. Given the quick growth of the techniques for quantum error mitigation (you can read an overview in the `Users Guide <https://mitiq.readthedocs.io/en/stable/guide/guide-error-mitigation.html>`_ ), it is natural to update the toolchain with new techniques and features. If you'd like to propose a new feature, please open an issue with the `feature request <https://github.com/unitaryfund/mitiq/issues/new/choose>`_ template on Github.
+
+.. _contributing_research_features:
+
+-------------------------
+Contributing new features
+-------------------------
+If you have some code or have a plan to add it to Mitiq, discuss it in an issue and once ready to upload the code, review the `contributing <contributing.html>`_ guidelines for the steps to take.
```

Comment:
"If you have some code"? Maybe "If you'd like to contribute new features to Mitiq, ..."?